wolfcrypt/src/error.c

```diff
@@ -434,6 +434,12 @@ const char* wc_GetErrorString(int error)
     case ECC_PRIVATEONLY_E:
         return "Invalid use of private only ECC key";
 
+    case WC_CAAM_E:
```

Comment:
Can we use a generic crypto hardware timeout / wait enum name here? Something like WC_HW_E and WC_HW_TIMEOUT_E?